bin/combine-pot-files.php

```diff
@@ -0,0 +1,104 @@
+<?php
+/**
+ * Command line script for merging two .pot files.
+ *
+ * @package WooCommerce Admin
+ */
+
+/**
+ * Get the two file names from the command line.
+ */
+if ( $argc < 2 ) {
+	echo "Usage: php -f {$argv[0]} source-file.pot destination-file.pot\n";
+	exit;
+}
+
+for ( $index = 1; $index <= 2; $index++ ) {
+	if ( ! is_file( $argv[ $index ] ) ) {
+		echo "File not found: {$argv[ $index ]}\n";
+		exit;
+	}
+}
+
+/**
+ * Check whether an output locale has been requested.
+ */
+if ( isset( $argv[3] ) && 0 === stripos( $argv[3], 'lang=' ) ) {
```

Comment:
Nice 👍

Reply:
> Out of curiosity, how will translations be managed going forward?

If the translator is using GlotPress then the existing strings & translations are stored in the DB and there is a feature for updating to a new version of the strings. 

For other tools I suspect the translators copy the source file to another location since translation files are always replaced in plugin/theme updates.